### ISSUE TITLE
ReadTheBuild build failing with AttributeError: 'module' object has no attribute 'UUIDField'

### ISSUE THREAD

**Issue by alexlittle** [mPowering/django-orb#441]
From StackOverflow issue could be related to mutual imports: http://stackoverflow.com/questions/1250103/attributeerror-module-object-has-no-attribute#1250135 - though I can;t see that this is the case here.

Full stacktrace from RTD:

 Running Sphinx v1.3.5
making output directory...

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/mpowering/envs/latest/local/lib/python2.7/site-packages/sphinx/cmdline.py", line 243, in main
    opts.warningiserror, opts.tags, opts.verbosity, opts.jobs)
  File "/home/docs/checkouts/readthedocs.org/user_builds/mpowering/envs/latest/local/lib/python2.7/site-packages/sphinx/application.py", line 127, in __init__
    confoverrides or {}, self.tags)
  File "/home/docs/checkouts/readthedocs.org/user_builds/mpowering/envs/latest/local/lib/python2.7/site-packages/sphinx/config.py", line 277, in __init__
    execfile_(filename, config)
  File "/home/docs/checkouts/readthedocs.org/user_builds/mpowering/envs/latest/local/lib/python2.7/site-packages/sphinx/util/pycompat.py", line 128, in execfile_
    exec_(code, _globals)
  File "/home/docs/checkouts/readthedocs.org/user_builds/mpowering/envs/latest/local/lib/python2.7/site-packages/six.py", line 672, in exec_
    exec("""exec _code_ in _globs_, _locs_""")
  File "<string>", line 1, in <module>
  File "conf.py", line 22, in <module>
    django.setup()
  File "/home/docs/checkouts/readthedocs.org/user_builds/mpowering/envs/latest/local/lib/python2.7/site-packages/Django-1.7.11-py2.7.egg/django/__init__.py", line 21, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/docs/checkouts/readthedocs.org/user_builds/mpowering/envs/latest/local/lib/python2.7/site-packages/Django-1.7.11-py2.7.egg/django/apps/registry.py", line 108, in populate
    app_config.import_models(all_models)
  File "/home/docs/checkouts/readthedocs.org/user_builds/mpowering/envs/latest/local/lib/python2.7/site-packages/Django-1.7.11-py2.7.egg/django/apps/config.py", line 202, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/docs/checkouts/readthedocs.org/user_builds/mpowering/checkouts/latest/docs/../orb/models.py", line 44, in <module>
    class Resource(TimestampBase):
  File "/home/docs/checkouts/readthedocs.org/user_builds/mpowering/checkouts/latest/docs/../orb/models.py", line 68, in Resource
    guid = models.UUIDField(null=True, default=uuid.uuid4, unique=True, editable=False)
AttributeError: 'module' object has no attribute 'UUIDField'

Exception occurred:
  File "/home/docs/checkouts/readthedocs.org/user_builds/mpowering/checkouts/latest/docs/../orb/models.py", line 68, in Resource
    guid = models.UUIDField(null=True, default=uuid.uuid4, unique=True, editable=False)
AttributeError: 'module' object has no attribute 'UUIDField'
The full traceback has been saved in /tmp/sphinx-err-GQoEx4.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!

**Comment by bennylope** [comment]
I created a project on Read the Docs from the wellfire repository and got a successful build. I don't have access to the [mpowering.rtfd.org](https://readthedocs.org/projects/mpowering/) admin interface, but here are the settings I used:

- NOT checked: Install your project inside a virtualenv using setup.py install
- Requirements file: `requirements.txt`
- Single version: NOT checked checked (this should have no effect on the bug)
- Python configuration file: `docs/conf.py` checked (this should have no effect on the bug)
- Default branch: left blank checked (this should have no effect on the bug)
- Default version: latest checked (this should have no effect on the bug)
- PDF and EPUB: NOT checked (this should have no effect on the bug)
- IS checked: "Give the virtual environment access to the global site-packages dir."

I've added you as a maintainer to that RTFD project to see and experiment.

Planning on removing that project after resolving this issue - no sense in having duplicate documentation sites!

**Also, if you [add me as a maintainer](https://readthedocs.org/dashboard/mpowering/users/) (username: @bennylope) on the RTFD project then I can just take a look myself.**
